Sources/SwiftProtobuf/Google_Protobuf_Any.swift

```diff
@@ -545,12 +556,12 @@ public struct Google_Protobuf_Any: Message, _MessageImplementationBase, _ProtoNa
                 if let messageType = Google_Protobuf_Any.wellKnownTypes[messageTypeName] {
                     let m = try messageType.init(unpackingAny: self)
                     let value = try m.jsonString()
-                    return "{\"@type\":\"\(type(of: m).anyTypeURL)\",\"value\":\(value)}"
+                    return "{\"@type\":\"\(typeURL)\",\"value\":\(value)}"
```

Comment:
This looks reasonable to me, but I don't know enough about `Any` to make a good judgment, probably.